src/features/filter.ts

```diff
@@ -1,10 +1,60 @@
-export const actions = { /* put action creators here */};
+import { Status } from '../types/Status';
 
-const filterReducer = () => {
+const SET_STATUS = 'filter/SET_STATUS';
+const SET_QUERY = 'filter/SET_QUERY';
+
+type SetStatus = {
+  type: 'filter/SET_STATUS',
```

Comment:
```suggestion
  type: SET_STATUS,
```

Same for the rest of actions/reducers